README.md

```
@@ -520,6 +520,8 @@ cd seaweedfs/weed && make install
 
 Once this is done, you will find the executable "weed" in your `$GOPATH/bin` directory
 
+How to run with docker and more here: https://github.com/seaweedfs/seaweedfs/wiki/Getting-Started
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)